dev-db/postgresql/postgresql-9.6.10.recipe

```diff
@@ -0,0 +1,160 @@
+	"
+
+PROVIDES_server="
+	cmd:clusterdb$secondaryArchSuffix = $portVersion compat >= 9
+	cmd:createdb$secondaryArchSuffix = $portVersion compat >= 9
+	cmd:createlang$secondaryArchSuffix = $portVersion compat >= 9
+	cmd:createuser$secondaryArchSuffix = $portVersion compat >= 9
+	cmd:dropdb$secondaryArchSuffix = $portVersion compat >= 9
+	cmd:droplang$secondaryArchSuffix = $portVersion compat >= 9
+	cmd:dropuser$secondaryArchSuffix = $portVersion compat >= 9
+	cmd:ecpg$secondaryArchSuffix = $portVersion compat >= 9
+	cmd:initdb$secondaryArchSuffix = $portVersion compat >= 9
+	cmd:pg_archivecleanup$secondaryArchSuffix = $portVersion compat >= 9
+	cmd:pg_basebackup$secondaryArchSuffix = $portVersion compat >= 9
+	cmd:pg_config$secondaryArchSuffix = $portVersion compat >= 9
+	cmd:pg_controldata$secondaryArchSuffix = $portVersion compat >= 9
+	cmd:pg_ctl$secondaryArchSuffix = $portVersion compat >= 9
+	cmd:pg_dump$secondaryArchSuffix = $portVersion compat >= 9
+	cmd:pg_dumpall$secondaryArchSuffix = $portVersion compat >= 9
+	cmd:pg_isready$secondaryArchSuffix = $portVersion compat >= 9
+	cmd:pg_receivexlog$secondaryArchSuffix = $portVersion compat >= 9
+	cmd:pg_resetxlog$secondaryArchSuffix = $portVersion compat >= 9
+	cmd:pg_recvlogical$secondaryArchSuffix = $portVersion compat >= 9
+	cmd:pg_restore$secondaryArchSuffix = $portVersion compat >= 9
+	cmd:pg_rewind$secondaryArchSuffix = $portVersion compat >= 9
+	cmd:pg_test_fsync$secondaryArchSuffix = $portVersion compat >= 9
+	cmd:pg_test_timing$secondaryArchSuffix = $portVersion compat >= 9
+	cmd:pg_upgrade$secondaryArchSuffix = $portVersion compat >= 9
+	cmd:pg_xlogdump$secondaryArchSuffix = $portVersion compat >= 9
+	cmd:pgbench$secondaryArchSuffix = $portVersion compat >= 9
+	cmd:postgres$secondaryArchSuffix = $portVersion compat >= 9
+	cmd:postmaster$secondaryArchSuffix = $portVersion compat >= 9
+	cmd:reindexdb$secondaryArchSuffix = $portVersion compat >= 9
+	cmd:vacuumdb$secondaryArchSuffix = $portVersion compat >= 9
+	"
+REQUIRES_server="
+	haiku$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	postgresql${secondaryArchSuffix} == $portVersion base
+	"
+
+PROVIDES_devel="
+	postgresql${secondaryArchSuffix}_devel = $portVersion
+	devel:libecpg$secondaryArchSuffix = $portVersion
+	devel:libecpg_compat$secondaryArchSuffix = $portVersion
+	devel:libpgcommon$secondaryArchSuffix = $portVersion
+	devel:libpgport$secondaryArchSuffix = $portVersion
+	devel:libpgtypes$secondaryArchSuffix = $portVersion
+	devel:libpq$secondaryArchSuffix = $portVersion
+	"
+REQUIRES_devel="
+	postgresql${secondaryArchSuffix} == $portVersion base
+	postgresql${secondaryArchSuffix}_server == $portVersion
+	"
+
+if [ -n "$secondaryArchSuffix" ]; then
+	CONFLICTS_server="
+		postgresql_server
+		"
+	CONFLICTS_devel="
+		postgresql_devel
+		"
+fi
+
+BUILD_REQUIRES="
+	devel:libreadline$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	haiku${secondaryArchSuffix}_devel
```

Comment:
Suggestion, maybe:
```diff
-if [ -n "$secondaryArchSuffix" ]; then
-	CONFLICTS_server="
-		postgresql_server
-		"
-	CONFLICTS_devel="
-		postgresql_devel
-		"
-fi
 BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
 	devel:libreadline$secondaryArchSuffix
 	devel:libz$secondaryArchSuffix
 	"
 BUILD_PREREQUIRES="
-	haiku${secondaryArchSuffix}_devel
 	cmd:aclocal
```

I might be wrong, but I think we might not need any `CONFLICTS*`. After all, we're not defining `commandBinDir` so there won't be any conflicting files among the `lib:*` and `cmd:*`.
Are there other files (e.g. in `$dataDir`) which would have different contents for primary vs secondary arch?

Hmm, maybe I should build the recipe before of asking :)

Reply:
> I might be wrong, but I think we might not need any CONFLICTS*. After all, we're not defining commandBinDir so there won't be any conflicting files among the lib:* and cmd:*.

One shouldn't have two copies of the same Postgres server installed at once, I believe. I'll add a comment.